tidy3d/components/simulation.py

```diff
@@ -3467,7 +3478,10 @@ def warn(structure, istruct, side) -> None:
                             and struct_val > sim_val
                             and abs(sim_val - struct_val) < lambda0 / 2
                         ):
-                            warn(structure, istruct, axis + "-min")
+                            if boundary[0].extrude_structures:
+                                warn_extruded(structure, istruct, axis + "-min")
+                            else:
+                                warn(structure, istruct, axis + "-min")
```

Comment:
what if a structure less than half wavelength away from pml, but still not close enough to be extruded?